package.json

```diff
@@ -31,8 +30,14 @@
   "dependencies": {
     "cbor": "^3.0.3",
     "commander": "^2.11.0",
+    "composer-admin": "^0.19.0",
```

Comment:
`js-yaml` should be added to the dependencies, since it's a Caliper-level dep.

The platform-specific dependencies should not be saved to `package.json`:
* `composer-*`
* `fabric-*`
* `grpc`
* `protocol-buffers`
* `sawtooth-sdk`

Reply:
ok, i will do it.